src/Init/SimpLemmas.lean

```diff
@@ -152,7 +153,7 @@ theorem Bool.or_assoc (a b c : Bool) : (a || b || c) = (a || (b || c)) := by
 @[simp] theorem bne_self_eq_false' [DecidableEq α] (a : α) : (a != a) = false := by simp [bne]
 
 @[simp] theorem Nat.le_zero_eq (a : Nat) : (a ≤ 0) = (a = 0) :=
-  propext ⟨fun h => Nat.le_antisymm h (Nat.zero_le ..), fun h => by simp [h]⟩
+  propext ⟨fun h => Nat.le_antisymm h (Nat.zero_le ..), fun h => by simp [h]; decide⟩
```

Comment:
and here, and etc.

Reply:
This one failed with:

```
Init/SimpLemmas.lean:156:84: error: unknown constant 'Lean.Meta.Simp.Config'
```

Any suggestions/workarounds?

Reply:
I replaced `simp` by `rw`

Reply:
No, this is some staging issue. Maybe we can ask @Kha or @digama0 for assistance on this, I still don't understand how to do these updates.

Please revert changes you've made to avoid the `unknown constant 'Lean.Meta.Simp.Config'` changes.

Reply:
This is an interesting edge case. We indeed will not be able to use `simp` config options this early in `Init`, because we have to elaborate the simp config and to do that we need `Lean.Meta.Simp.Config` to exist in the environment. We might be able to move it earlier but dependency order in Init is challenging. IMO `simp; decide` is a reasonable alternative solution since this issue only exists for a few proofs in Init.